Source/Details/_ASDisplayLayer.mm

```diff
@@ -179,13 +138,14 @@ + (id)defaultValueForKey:(NSString *)key
 {
   if ([key isEqualToString:@"displaysAsynchronously"]) {
     return @YES;
+  } else if ([key isEqualToString:@"opaque"]) {
```

Comment:
@appleguy In my testing, CALayer only calls this method once per class and caches the result, and it also correctly handles overriding standard layer properties as far as I can tell. So the overall performance is probably better with this implementation, which for me overrides the point about a user understanding where an unexpected opacity came from. Thoughts?